koku/masu/database/trino_sql/reporting_ocpawscostlineitem_daily_summary.sql

```diff
@@ -399,18 +399,22 @@ SELECT
     {{ocp_source_uuid}} as ocp_source,
     {{year}} as year,
     {{month}} as month
-FROM aws_line_items as aws
+FROM hive.{{schema | sqlsafe}}.aws_line_items as aws
 INNER JOIN cte_ocp_filtered_resources as ocpaws
     ON aws.lineitem_resourceid = ocpaws.resource_id
     AND DATE(aws.lineitem_usagestartdate) = ocpaws.usage_start
 WHERE aws.year = {{year}}
 AND aws.month = {{month}}
+AND aws.source = {{aws_source_uuid}}
 group by aws.lineitem_resourceid, ocpaws.usage_start
 {% endif %}
 ;
 
+
+{% if not unattributed_storage %}
```

Comment:
An explicit boolean check would be safer here since any truthy value would evaluate to `True`. 
```suggestion
{% if unattributed_storage is not true %}
```

```shell
In [45]: data = {'unattributed_storage': 'yo'}

In [46]: template = '{% if unattributed_storage is true %}Storage SQL{% endif %}'

In [47]: JinjaSql().prepare_query(template, data)
Out[47]: ('', {})

In [48]: template = '{% if unattributed_storage %}Storage SQL{% endif %}'

In [49]: JinjaSql().prepare_query(template, data)
Out[49]: ('Storage SQL', {})
```

Reply:
This is a sql_param we are passing in so its probably fine, its also consistent with how we do it in our other sql files.